stacks_queues/queue.py

```diff
@@ -15,22 +23,30 @@ def __init__(self):
         self.front = -1
         self.rear = -1
         self.size = 0
-      
-
+        
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
-        """
```

Comment:
⚠️  This doesn't work as a circular buffer.  I suggest using the `self.front` and `self.rear` indexes.